### PR TITLE
better integration with cime

### PR DESCRIPTION
**Pull request by jedwards4b**
I will be creating templates for various postprocessing steps which are better integrated with cime.
The original files are still in place and so current functionality is not adversely affected.

### CODE REVIEW

File: timeseries/template.timeseries [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+
+if __name__ == "__main__":
+    _main_func(__doc__)
+
+
+
+
+#mpiexec_mpt dplace -s 1 ./cesm_tseries_generator.py --debug 0  --caseroot /glade/u/home/jedwards/sandboxes/cesm2_1/cime.two/scripts/kensemble.001/postprocess >> ${log_filename} 2>&1
```
Comment (bertinia) [inline timeseries/template.timeseries]: @jedwards4b - I don't think you want this path hardcoded in the template file. 
It should be a variable pass into the jinja2 rendering call.

Comment (jedwards4b) [reply timeseries/template.timeseries]: That line is in a comment and is not at all relevant to the code.   I will remove it.

**Comment by jedwards4b** [comment]
That's correct, all I've done here is added a script 'template.timeseries' that will interact with a case object and removed some hard coded paths.  If there is no case object then the template is ignored and the old timeseries script is used.  All previous functionality is maintained.

**Comment by bertinia** [comment]
@jedwards4b - please test your changes on your branch using actual data before issuing a PR. In this case, I accepted this PR but the timeseries failed due to a change that you made to pass back the 
CLA options. Namely, debug needed to be passed as options.debug[0]. Thanks.

**Pull request merged**